go/libkb/ca.go

```diff
@@ -3,10 +3,35 @@
 
 package libkb
 
-var BundledCAs = map[string]string{
-	"api.keybase.io":          apiCA,
-	"gregord.kbfs.keybase.io": KBFSProdCA,
-	"gregord.dev.keybase.io":  KBFSDevCA,
+import "strings"
+
+var apiCAOverrideForTest = ""
+
+// GetBundledCAsFromHost returns in root CA in []byte for given host, or nil if
+// no matching CA is found for host.
+func GetBundledCAsFromHost(host string) (rootCA []byte, ok bool) {
+	host = strings.TrimSpace(strings.ToLower(host))
+	switch {
+	case host == "api.keybase.io":
+		if len(apiCAOverrideForTest) > 0 {
+			return []byte(apiCAOverrideForTest), true
+		}
+		return []byte(apiCA), true
+
+	// Staging CA.
+	case strings.HasSuffix(host, "dev.keybase.io"),
+		strings.HasSuffix(host, "dev.keybaseapi.com"):
+		return []byte(KBFSDevCA), true
+
+	// Prod CA.
+	case strings.HasSuffix(host, "kbfs.keybase.io"),
```

Comment:
Needs `core.keybase.io` right?

Reply:
good catch; will update!

Reply:
actually it seems core.keybase.io uses the CA for api.keybase.io. Is that correct?

Reply:
not for gregor. this was our major confusion

Reply:
Could be a problem with this change then, since Gregor could be on that address and will be using KBFS CA.

Reply:
I'll just bundle both CA in then.